client/src/components/Search/Filters/FilterList.js

```diff
@@ -167,17 +169,26 @@ class Filters extends React.Component {
         if (event.key === 'Enter') this.handleSearch();
     }
 
-    render() {
+    handleTempChange(value){
+        var updatedState = this.state.filterList.hideTemporary
+        updatedState.params.hideTemporary = value
+        this.setState({
+            ...this.state,
+            ...updatedState
+        })
+        this.handleSearch()
+    }
 
-        const { decades } = this.props
+    render() {
         const { showFilters, filterList} = this.state;
         const expandIcon = showFilters ? "remove" : "add";
         let dateMap = (
             <DateFilter 
                 onRemove={() => this.clearTags('date')}
                 data={filterList.date}
-                decades={decades}
                 filterMode={filterList.date.config.filterMode}
+                hideTemporary={filterList.hideTemporary.params.hideTemporary}
+                onTempChange={(value) => this.handleTempChange(value)}
```

Comment:
nit:  just writing `onTempChange={this.handleTempChange}` will do the same thing